bpf/lib/encap.h

```diff
@@ -32,6 +34,9 @@ __encap_with_nodeid(struct __ctx_buff *ctx, __u32 src_ip, __be16 src_port,
 	if (seclabel == HOST_ID)
 		seclabel = LOCAL_NODE_ID;
 
+	if (seclabel > ENCAP_MAX_VNI)
+		return DROP_INVALID_IDENTITY;
```

Comment:
What do we do if we see these drops? Does it signal that the `seclabel` (security identity) of the workload is somehow incorrect?

Reply:
I would consider it as indication of a code bug.

Either the agent selecting a bad identity for the workload. Or the datapath trying to forward an non-eligible identity - I've caught myself wanting to forward a source identity in the N/S loadbalancer path, but not considering CIDR identities.